handlers/auth_code_request_create.go

```diff
@@ -120,6 +122,16 @@ func validateCorporateBody(req *http.Request, authCodeReqSvc *service.AuthCodeRe
 		return false, err
 	}
 
+	// Check whether user has made recent filings
+	userMultipleRequests, err := authCodeReqSvc.CheckMultipleUserSubmissions(email)
```

Comment:
I would perhaps change this to `userExceededRequests`, and the same in the log statement below. As a user can submit multiple requests (up to 3), they just can't exceed it, and it might cause confusion.